### ISSUE TITLE
The back button does not work in LibreTube

### ISSUE THREAD

**Issue by Pamilg8** [libre-tube/LibreTube#5649]
### Steps to reproduce

1. Enter LibreTube
2. Press the back button on your device
3. Back button doesn't work. It does not work in any section

### Expected behavior

The back button should work without problems 

### Actual behavior

Back button functionality is not working in the app. 
This problem happened suddenly 

### LibreTube version

0.21.1 Debug

### Android version

Android 12

### Other details

https://github.com/libre-tube/LibreTube/assets/149590337/b98c6b98-d005-46b8-ae1a-0d3f93f0ce60

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
Good catch, it seems like some dependency update now forces us to explicitly enable handling back presses - probably due to the new back gesture invocation APIs added in Android 14.

**Comment by Pamilg8** [comment]
> Good catch, it seems like some dependency update now forces us to explicitly enable handling back presses - probably due to the new back gesture invocation APIs added in Android 14.

I tried the debug version and it didn't fix the problem

**Comment by Bnyro** [comment]
Strange, it's fixed for me.

**Comment by Pamilg8** [comment]
> Strange, it's fixed for me.

Can you please keep the issue open? Maybe someone helped me😔

**Comment by Bnyro** [comment]
> > Strange, it's fixed for me.
> 
> tried the latest commits but it isn't fixed. It works though by reverting this commit [ce8ed66](https://github.com/libre-tube/LibreTube/commit/ce8ed66e927ab2232eb29f756692e80523b34ea3)
>
Reverting this commit only or using a build before that commit (so also reverting any other changes)?

**Comment by Pamilg8** [comment]
I test LibreTube on different devices.( Android 12, One UI Core 4.1)
Or Android 13, One UI 5.1

**Comment by akano12** [comment]
Can't reproduce either.
Samsung S20 A13, can you try latest debug?

**Comment by Pamilg8** [comment]
> Can't reproduce either. Samsung S20 A13, can you try latest debug?

I have tried the latest debug on samsung a32 and samsung a21s, a31 and the problem is not fixed. Samsung a13 runs full One UI. You try A21s and a12 running One UI Core version. A32 and a31 are running full One UI 5.1 but they have the same problem

**Comment by Pamilg8** [comment]
1709460796.079 10696 21087 21087 I ViewRootImpl@f363c38[Default]: ViewPostIme key 0
1709460796.166 10696 21087 21087 I ViewRootImpl@f363c38[Default]: ViewPostIme key 1
1709460800.235 10696 21087 21087 I ViewRootImpl@f363c38[Default]: MSG_WINDOW_FOCUS_CHANGED 0 1
1709460800.264 10696 21087 21087 D InputTransport: Input channel destroyed: 'ClientS', fd=140
